### PR TITLE
Techno Slide Whistle: We've Come A Long Way

### PR DESCRIPTION
**Pull request by Brycey92**
Not done yet, but FAR more functional than before this PR

### CODE REVIEW

File: main/modes/mode_slide_whistle.c [hunk 1]
```diff
@@ -1031,14 +1141,31 @@ void  slideWhistleExitMode(void)
...
+ *
+ * @param evt The button event that occurred
+ */
+void  slideWhistleTouchCallback(touch_event_t* evt)
+{
+    slideWhistle->touchHeld = evt->state != 0;
+    slideWhistle->shouldPlay = evt->state != 0 || slideWhistle->aHeld;
+
+    //slideWhistle->touchPosition = roundf((evt->position * BAR_X_WIDTH) / 255);
+    getTouchCentroid(&slideWhistle->touchPosition, &slideWhistle->touchIntensity);
```
Comment (AEFeinstein) [inline main/modes/mode_slide_whistle.c:1163]: I thought that calling this from the call gave choppy results?

Comment (AEFeinstein) [reply main/modes/mode_slide_whistle.c:1163]: Might be better if you call this either in the main loop or the accel callback

**Review by AEFeinstein — APPROVED** [review]

File: main/modes/mode_slide_whistle.c (hunk 1 above)
Comment (AEFeinstein) [reply main/modes/mode_slide_whistle.c:1163]: Fixed this myself

Comment (Brycey92) [reply main/modes/mode_slide_whistle.c]: Thanks! Meant to get around to it today, but you beat me to it :)

**Pull request merged**